site/source/docs/api_reference/proxying.h.rst

```diff
@@ -72,19 +72,19 @@ Functions
 
   Signal the end of a task proxied with ``emscripten_proxy_sync_with_ctx``.
 
-.. c:function:: int emscripten_proxy_async(em_proxying_queue* q, pthread_t target_thread, void (\*func)(void*), void* arg)
+.. c:function:: int emscripten_proxy_async(em_proxying_queue* q, pthread_t target_thread, void (*func)(void*), void* arg)
```

Comment:
Does the leaving `*` still appear in the rendered text?  (with the current/older version of sphinx?)

Reply:
It does appear with my current version (4.5.0). Can someone check if this is the case with the old version (2.4.4?), if they have the old version installed locally?

Reply:
OK I installed 2.4.4 with pip install -r requirements-dev.txt` and `*` appears in there too.